city_scrapers/spiders/il_arts_council.py

```diff
@@ -0,0 +1,129 @@
+import unicodedata
+from datetime import datetime as dt
+
+from city_scrapers_core.constants import BOARD
+from city_scrapers_core.items import Meeting
+from city_scrapers_core.spiders import CityScrapersSpider
+
+
+class IlArtsCouncilSpider(CityScrapersSpider):
+    name = "il_arts_council"
+    agency = "Illinois Arts Council"
+    timezone = "America/Chicago"
+    start_urls = ["http://www.arts.illinois.gov/about-iac/governance/council-meetings"]
+
+    def parse(self, response):
+        """
+        `parse` should always `yield` Meeting items.
+
+        Change the `_parse_title`, `_parse_start`, etc methods to fit your scraping
+        needs.
+        """
+        for table in response.xpath('//table/tbody'):
+            year = self._get_year(table)
+            for item in table.xpath('./tr')[1::]:
+                meeting = Meeting(
+                    title=self._parse_title(item),
+                    description=self._parse_description(item),
+                    classification=self._parse_classification(item),
+                    start=self._parse_start(item, year),
+                    end=self._parse_end(item),
+                    all_day=self._parse_all_day(item),
+                    time_notes=self._parse_time_notes(item),
+                    location=self._parse_location(item),
+                    links=self._parse_links(item),
+                    source=self._parse_source(response),
+                )
+
+                meeting["status"] = self._get_status(meeting)
+                meeting["id"] = self._get_id(meeting)
+
+                yield meeting
+
+    def _parse_title(self, item):
+        """Parse or generate meeting title."""
+        return "Board Meeting"
```

Comment:
"Agency Board" might be the best default here. We try to leave "Meeting" out if we can